packages/twenty-front/src/modules/settings/security/graphql/queries/getApprovedAccessDomains.ts

```diff
@@ -0,0 +1,12 @@
+import { gql } from '@apollo/client';
+
+export const GET_ALL_APPROVED_ACCESS_DOMAINS = gql`
+  query GetAllApprovedAccessDomains {
+    getAllApprovedAccessDomains {
```

Comment:
I don't think GetAll is a common naming pattern. getApprovedAccessDomains ? (we even should probably have do just ApprovedAccessDomains but that's not the case in many cases so up to you between the two... indexMetadatas, objects, fields etc are named the right way and the many other getXXX are not imo)